src/main/webapp/app/course/learning-paths/participate/learning-path-graph-sidebar.component.html

```diff
@@ -0,0 +1,36 @@
+<div class="learning-path-sidebar">
+    <div [hidden]="collapsed" class="expanded-graph">
+        <div class="card">
+            <!-- header -->
+            <div class="card-header d-flex bg-primary text-white" [ngbTooltip]="'artemisApp.learningPath.sideBar.hide' | artemisTranslate" (click)="collapsed = true">
+                <h3 class="card-title">
+                    <fa-icon [icon]="faNetworkWired"></fa-icon>
+                    <span class="ms-2">{{ 'artemisApp.learningPath.sideBar.header' | artemisTranslate }}</span>
+                </h3>
+                <div class="col-auto">
+                    <fa-icon [icon]="faChevronLeft"></fa-icon>
+                </div>
+            </div>
+            <!-- body -->
+            <div class="card-body">
+                <jhi-learning-path-graph
+                    #learningPathGraphComponent
+                    *ngIf="learningPathId"
+                    [courseId]="courseId"
+                    [learningPathId]="learningPathId"
+                    (nodeClicked)="nodeClicked.emit($event)"
```

Comment:
Why do you emit a on click event if a on click event is emitted?

Reply:
This just passes the event (received from the child) to the parent.